README.md

```diff
@@ -86,6 +86,27 @@ If you have any questions about Zodiac, join the [Gnosis Guild Discord](https://
 
 The [Zodiac documentation](https://gnosis.github.io/zodiac/) offers tutorials on how to use the Zodiac App, and detailed developer resources on how to build your own Zodiac module, modifier, or guard will be available soon.
 
+### Zodiac Banners and Badges
+Show your support for Zodiac by including our "Built By Zodiac" badges or banners on your site or in your repository. Check out the previews below and download the ones that you like best. Please wrap the image with a link to [Zodiac](https://gnosis.github.io/zodiac/).
+
+NOTE: Copy the image url and use the hosted version to ensure it's always up-to-date.
+
+**Banners**
+
+![White on Black Banner](https://raw.githubusercontent.com/gnosis/zodiac/654afcfdc453027ff1d8fa8117bd6648a8d2034c/branding/zodiac-banner-black-white.svg)
```

Comment:
All these links should be changed to reflect the URL where these files will live eventually, which will allow us to update them and have the changes show everywhere else.

for example
`https://raw.githubusercontent.com/gnosis/zodiac/master/branding/zodiac-badge-black-white.svg`